### DISCUSSION TITLE
How to add a PlaceNode to a camera?

### DISCUSSION THREAD

**Discussion by MRdesperado1** [pelicanmapping/osgearth#1847] [Support Q&A]
Hello, there was a problem when I used osgEarth to create another camera.
How to add a PlaceNode to a camera?
I used the camera to create a thumbnail, just like in a large game. I expect some key points to be marked on the thumbnail, such as moving aircraft, etc. However, my PlaceNode cannot be displayed on the thumbnail, but when I use Root to add these created PlaceNode, they can be displayed normally on the big map. What is the reason?
The code for creating PlaceNode is as follows：

> osgEarth::Style styleairfly;
> 	osgEarth::Symbology::TextSymbol* textstyle2 = styleairfly.getOrCreateSymbol<osgEarth::Symbology::TextSymbol>();
> 	 styleairfly.getOrCreate<osgEarth::Symbology::TextSymbol>()->encoding() = osgEarth::Symbology::TextSymbol::ENCODING_UTF8;
> 	textstyle2->fill()->color() = osg::Vec4f(1.0, 0.0, 0.0, 0.5);
> 	textstyle2->halo()->color() = osg::Vec4f(1.0, 0.8, 0.8, 1.0);
> 	textstyle2->font() = "simsun.ttc";
> 	textstyle2->size() = 16.0;
> 	textstyle2->alignment() = osgEarth::Symbology::TextSymbol::ALIGN_CENTER_BOTTOM;
> ffLabel = new osgEarth::Annotation::PlaceNode(osgEarth::GeoPoint(mapNode->getMapSRS(), osg::Vec3d(109.1347, 34.3834, 100)), transcode::transferunicode("F35C-1"), styleairfly, jichang);

The code for creating Camera is as follows：
	

> radar = new osg::Camera;
> 	radar->setName("radar");
> 	radar->setClearColor(osg::Vec4(0.2, 0.2, 0.2, 1.0));
> 	radar->setRenderOrder(osg::Camera::POST_RENDER);
> 	radar->setAllowEventFocus(false);
> 	radar->setClearMask(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
> 	radar->setReferenceFrame(osg::Transform::ABSOLUTE_RF);
> 	radar->setViewport(0,0,256,256);
> 	radar->setProjectionMatrixAsPerspective(30.0f, static_cast<double>(256) / static_cast<double>(256), 1.0, 1000.0);

The code for adding placenode to camera is as follows：
      

>   radar->addChild(mapNode);
>         radar->addChild(ffLabel);
> 	mRoot->addChild(radar);

The effect after adding is shown in the figure：
![image](https://user-images.githubusercontent.com/38163799/140173276-cadc227a-0932-47f0-8987-37adb2a95646.png)
Add it directly to the big map, and the effect is as shown in the figure：
![image](https://user-images.githubusercontent.com/38163799/140173611-35185de3-88be-4d1d-a642-2bf9c1f9e8d4.png)
What is the reason?
Thank you very much. And I  want to ask you. My airport displays floating in the air. As the viewpoint moves, the airport will also move. How should this situation be solved?
I use OSG version 3.6.3 and osgearth version 2.10.

**Comment by gwaldron** [comment]
You might need to disable small feature culling on your inset camera:

`radar->setSmallFeatureCullingPixelSize(-1.0f);`

**Reply by MRdesperado1** [reply]
I tried to do as you said, but it didn't work. PlaceNode is still not displayed. Is there any other way?
![image](https://user-images.githubusercontent.com/38163799/141120162-1a4d064e-eb10-42f4-ac78-1dcb02ae283e.png)
